test/TestServerProjectsLowLevel/dpg-customization/Generated/DPGClient.cs

```diff
@@ -0,0 +1,429 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Core;
+using Azure.Core.Pipeline;
+
+namespace dpg_customization_LowLevel
+{
+    /// <summary> The DPG service client. </summary>
+    public partial class DPGClient
```

Comment:
nit: in .NET we use PascalCasing for client names. (See section 3.2.1 in the [.NET Framework Design Guidelines](https://learning.oreilly.com/library/view/framework-design-guidelines/9780135896457/ch03.xhtml)).

Reply:
This name comes from laguage.default.name of root or operation group in code model. We autorest don't deal with it when generating client. If we want PascalCasing, we need to fix in autorest. I created an issue https://github.com/Azure/autorest.csharp/issues/2008. Could you triage?

Reply:
We aren't concerned with following the guidelines in test code.